core/modules/filter/js/filter.js

```diff
@@ -104,6 +104,32 @@ Backdrop.filterEditorDetach = function(field, format, trigger) {
   }
 };
 
+/**
+ * Provides summary text for the "Formatting options" fieldset, under each
+ * textarea field with a text editor.
+ */
+Backdrop.behaviors.filterFieldsetSummaries = {
+  attach: function (context) {
+    $(context).find('fieldset.filter-wrapper').backdropSetSummary(function (element) {
+      var $summary = '';
```

Comment:
Convention is to only put a `$` in front of variables that contain jQuery objects. This `$summary` is a string, so it should just be `summary`.

Reply:
Done: [`69d0652` (#2550)](https://github.com/backdrop/backdrop/pull/2550/commits/69d065266f3651694d76fd6abc491af58b98071f)